2_process/src/process_data.R

```diff
@@ -0,0 +1,43 @@
+
+process_data <- function(){
+  
+  save_path <- "./2_process/out"
+  
+  # Prepare the data for plotting:
+  eval_data <- readr::read_csv(file.path("./1_fetch/out", 'model_RMSEs.csv'), col_types = 'iccd') %>%
```

Comment:
You can make the save path and input file path arguments to your function

Reply:
Good suggestion - I've implemented these changes to the process_data function.